### PR TITLE
Added 2 new files related to in-person; updated index pgs

### PR DESCRIPTION
**Pull request by hparkersd**
Added in-person sessions and also a supporting page for the VE List.  I can only see MarkDown so I will need to edit these again.  I also need to figure out how to add a few images.

### CODE REVIEW

File: content/docs/cve/cveIn-PersonExamProcess [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
```
Comment (n2ygk) [inline content/docs/cve/cveIn-PersonExamProcess:1]: I think you need to add a reference to this page in https://github.com/HamStudy/examtools-docs/blob/master/content/docs/cve/_index.md and maybe also https://github.com/HamStudy/examtools-docs/blob/master/content/docs/_index.md

File: content/docs/cve/cveVEList [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
```
Comment (n2ygk) [inline content/docs/cve/cveVEList:1]: As above, references to this page after to be added in the `_index.md` files.

**Review by n2ygk — CHANGES_REQUESTED** [review]
I believe you wanted to add screen shots as well. These go in https://github.com/HamStudy/examtools-docs/tree/master/content/docs/cve/images as png or jpeg files and then you reference them. See some existing pages with screenshots for examples.

**Comment by hparkersd** [comment]
I am at a stopping point.

**Comment by n2ygk** [comment]
> I am at a stopping point.

Are you ready for my review? I see you've just pushed some more...

**Comment by hparkersd** [comment]
> I believe you wanted to add screen shots as well. These go in https://github.com/HamStudy/examtools-docs/tree/master/content/docs/cve/images as png or jpeg files and then you reference them. See some existing pages with screenshots for examples.

I added an image.  That is all I have right now, but understand how to do it.

**Comment by n2ygk** [comment]
@hparkersd I made a couple of small fixes:
- replaced http: with https:
- replaced "hard coded" refs to docs.exam.tools with relrefs.

**Comment by hparkersd** [comment]
Thanks!  I was not able to get them to work in my new pages.  Was missing something.

**Pull request merged**